home.html

```diff
@@ -0,0 +1,112 @@
+      <div class="nav-bar">
+        <a href="home.html"><img class="click" src="images/logo.png" alt="logo"></a>
+        <div class="nav-links">
+          <a class="LINK" href="about.html">About</a>
+          <a class="LINK" href="#products">Our Products</a>
+          <a class="LINK"href="#Artists">Artists</a>
+          <a href="" class="flawless">FLAWLESS</a>
+        </div>
+      </div>
+      <i class="fa fa-bars fa-2x" id="bars"></i>
+    </nav>
+    <div class="nav-menu">
+      <i class="fa fa-times fa-2x" id="closingIcon"></i>
+      <ul>
+        <li class="mobileLinks"><a href="home.html">Home</a></li>
+        <li class="mobileLinks"><a href="#products">Products</a></li>
+        <li class="mobileLinks"><a href="about.html">About</a></li>
+      </ul>
+    </div>
+    <div class="main">
+      <q>Hi there!</q>
+      <h1>With Flawless, Glow like never before!</h1>
+      <p class="description">
+        Looking young and beautiful never comes without using the best cosmetics.
+        Here we have compiled the best skincare products by the best brands.
+      </p>
+      <p class="promotion">
+        10% OFF FOR FIRST PURCHASE
+      </p>
+      <a class="more" href="#viewmore">SEE OUR COLLECTIONS.</a>
+    </div>
+  </header>
+  <section class="products" id="products">
+    <h2>Main Products</h2>
+    <hr class="title-underline">
+    <div class="product-list">
+      <div class="skincare">
+        <img class="product-image" src="images/cleanser.png" alt="cleanser">
+        <h3>Cleansers</h3>
+        <p class="explication">Our cleansers help remove excess oil, makeup, sweat, dirt and dead skin.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/serum.png" alt="serum">
+        <h3>Serums</h3>
+        <p class="explication">By using our serums, your skin will be more radiant,full of colour and it'll be less dry.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/bodylotion.png" alt="lotion">
+        <h3>Lotions</h3>
+        <p class="explication">Usse our body lotions daily and you'll notice a difference in your skin's hydration levels.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/nails.png" alt="nails">
+        <h3>Nails</h3>
+        <p class="explication">Use our nail products to ensure that your nails are in good condition and the skin on hands and feet is well cared.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/hair.png" alt="hair"> 
+        <h3>Hair</h3>
+        <p class="explication">Our hair products will strengthen your hair follicles and enhance hair growth.</p>
+      </div>
+    </div>
+    <a class="button">View more products</a>
+  </section>
+  <div class="artists" id="Artists">
+    <h2 class="arts">Our Beautician Artists</h2>
+    <hr class="title-underline">
+    <div class="artist-wrapper"></div>
+    </div>
```

Comment:
***[Figma Design]***
![image](https://github.com/mariegrace31/capstone_one/assets/60083437/a51ea391-ad5b-4c55-a852-122dfdec03d1)

- [ ] You can see that there are background designs for every `images` in this section in the given design. As per the previous reviewer's suggestion kindly fix this to match the given design.